api/bybit/futures_client.py

```
@@ -563,28 +643,68 @@ def set_leverage(self, symbol: str, leverage: int) -> bool:
     def set_margin_mode(self, symbol: str, margin_mode: str) -> bool:
         """마진 모드 설정"""
         try:
+            # 계정 타입 자동 감지
+            account_type = self._detect_account_type()
```

Ruff (0.14.1)

Local variable `account_type` is assigned to but never used

Remove assignment to unused variable `account_type`

(F841)